src/utils.jl

```diff
@@ -94,3 +94,24 @@ Returns the unitless type of a (unitful) type or value. See Unitful.jl.
 i.e. `dropunits(1u"mm") == Int`
 """
 dropunits(v) = typeof(one(v))
+
+"""
+    householderbasis(n)
+
+Returns a pair of orthonormal tangent vectors `u` and `v` from a normal `n`, 
+such that `u`, `v`, and `n` form a right-hand orthogonal system.
+
+Reference: ["Tangent vectors to a 3-D surface normal: A geometric tool to find orthogonal vectors based on the Householder transformation"](https://doi.org/10.1016/j.cad.2012.11.003)
```

Comment:
Please use the reference format like in here: https://github.com/JuliaGeometry/Meshes.jl/blob/4d1d68cd43b0ac00c69395476053cc30296040a3/src/refinement/catmullclark.jl#L16-L20